src/components/Home.jsx

```diff
@@ -31,12 +34,49 @@ function Home() {
     }
   };
 
+  const handleTokenSubmit = async () => {
+    //create query for token collection in firestore
```

Comment:
Good comment - it shows something that's not obvious (that query call below is for firebase) to "casual reader"